logic/SituationManager.py

```diff
@@ -0,0 +1,47 @@
+import json
+import random
+
+from situation import Situation
+
+
+class SituationManager():
+    """SituationManager class"""
+
+    def __init__(self):
+        self.current_situation = None
+        self.situation_counter = 0
+
+    def load_situation(self) -> None:
+        """Loads a new situation"""
+        random.seed()
+
+        # Get the data for a landmark situation
+        if self.situation_counter % 5 == 0:
+            with open('situations/landmark.json', 'r') as file:
```

Comment:
Why not read both the json files in __init__ instead of reading them on every load?